messaging/propagate.go

```diff
@@ -17,6 +18,9 @@ func init() {
 	network.RegisterMessage(PropagateReply{})
 }
 
+// How long to wait before timing out on waiting for the time-out.
+const initialWait = 100000
```

Comment:
1)
> before timing out on waiting for the time-out.

Maybe just `before timeout` ?

2) What do you think about putting that as a exported variable instead ? Just proposing here but IIRC we had so many experiments requirings so many different timeouts because of long setup, overloaded nodes, etc that it may be useful to have it exported. If you don't think so, then I'll merge it like this, I just wanted to raise the point.